### PR TITLE
Feature/514 manage const values in cim xml converter

### PR DESCRIPTION
**Pull request by johevemi**
## Description

Manage const values in the converter.

## References

* #516

### CODE REVIEW

**Review by ZavezX — APPROVED** [review]

File: source/AggregationResultReceiver/source/AggregationResultReceiver.Application/Converters/ICimXmlConverter.cs [hunk 1]
```diff
@@ -18,13 +18,16 @@
 namespace Energinet.DataHub.Aggregations.AggregationResultReceiver.Application.Converters
 {
     /// <summary>
-    /// IMapToCimXml
+    /// Interface for CimXmlConverter
```
Comment (jonasdmoeller) [inline source/AggregationResultReceiver/source/AggregationResultReceiver.Application/Converters/ICimXmlConverter.cs:21]: The summary should describe what the interface is doing. The actual implementation is just a detail for users of the interface.
I.e. "Interface for converting list of aggregation results to CIM/XML documents"

File: source/AggregationResultReceiver/source/AggregationResultReceiver.Application/Helpers/IGuidGenerator.cs [hunk 2]
```diff
@@ -20,8 +20,9 @@ namespace Energinet.DataHub.Aggregations.AggregationResultReceiver.Application.H
     public interface IGuidGenerator
     {
         /// <summary>
-        /// Get a new Guid as a string with only digits
+        /// Create a new Guid as a string with only digits
         /// </summary>
+        /// <returns>string representation of a GUID without dashes</returns>
         public string GetGuidAsStringOnlyDigits();
```
Comment (jonasdmoeller) [inline source/AggregationResultReceiver/source/AggregationResultReceiver.Application/Helpers/IGuidGenerator.cs:26]: If method is creating a new guid, the maybe name it CreateGuidAs....

File: source/AggregationResultReceiver/source/AggregationResultReceiver.Application/IFileStore.cs [hunk 3]
```diff
@@ -23,13 +23,17 @@ namespace Energinet.DataHub.Aggregations.AggregationResultReceiver.Application
     public interface IFileStore
     {
         /// <summary>
-        /// Save converted message
+        /// Upload stream to blob store at given blob name
```
Comment (jonasdmoeller) [inline source/AggregationResultReceiver/source/AggregationResultReceiver.Application/IFileStore.cs:26]: The interface does not know about the implementation, so mentioning "blob store" in this context is not right. We could make a SqlFileStore implementation of the interface as well. Then the description would be wrong.

File: source/AggregationResultReceiver/source/AggregationResultReceiver.Application/IFileStore.cs [hunk 4]
```diff
@@ -23,13 +23,17 @@ namespace Energinet.DataHub.Aggregations.AggregationResultReceiver.Application
...
         /// </summary>
-        Task UploadConvertedMessageAsync(string fileName, Stream content);
+        /// <param name="blobName">Name of the blob</param>
+        /// <param name="content">Stream to upload</param>
+        Task UploadConvertedMessageAsync(string blobName, Stream content);
 
         /// <summary>
-        /// Fetch Blob
+        /// Download stream from blob store with given blob name
```
Comment (jonasdmoeller) [inline source/AggregationResultReceiver/source/AggregationResultReceiver.Application/IFileStore.cs:33]: Same as above

**Review by jonasdmoeller — CHANGES_REQUESTED** [review]

**Review by jonasdmoeller — APPROVED** [review]

**Pull request merged**